src/main/java/bio/terra/cli/context/GlobalContext.java

```diff
@@ -25,31 +22,38 @@
  * primarily for authentication and connection-related context values that will span multiple
  * workspaces.
  */
+@JsonAutoDetect(fieldVisibility = JsonAutoDetect.Visibility.ANY)
 public class GlobalContext {
```

Comment:
no. I wasn't trying to maintain backwards compatibility at all. perhaps worth a group discussion about when start treating this as more of a released product instead of a work in progress for tire-kicking. this PR is not backwards compatible in other ways though that would be harder to handle nicely (e.g. workspace is no longer tied to the current working directory like `git`), so regardless of what we decide I think this will have downstream effects.

Reply:
I think that's still good at this stage, just didn't see it in the top level description. Do users need to do anything, or will this just work by overwriting what's there?